ffg/emitter/yinyang_emitter.py

```diff
@@ -52,7 +52,7 @@ def emit_options(file, args):
     print("", file=file)
 
 
-def emit_function(tree: Operator, file):
+def emit_function(tree: Operator, file, is_wrapped=True):
```

Comment:
Maybe call this just wrap? is_wrapped sounds like it's a property of the tree being passed.

Reply:
done :)